net.krafting.PleasureDVR.yml

```diff
@@ -0,0 +1,130 @@
+app-id: net.krafting.PleasureDVR
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: PleasureDVR.py
+finish-args:
+  -  --share=ipc
```

Comment:
Add `--device=dri`

Reply:
Why is this needed exactly ? for networking ?

Reply:
OpenGL rendering